hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -201,6 +201,21 @@ public enum ChecksumCombineMode {
   // 3 concurrent stripe read should be enough.
   private int ecReconstructStripeReadPoolLimit = 10 * 3;
 
+  @Config(key = "ec.reconstruct.stripe.write.pool.limit",
+      defaultValue = "30",
+      description = "Thread pool max size for parallelly write" +
+          " available ec chunks to reconstruct the whole stripe.",
+      tags = ConfigTag.CLIENT)
+  private int ecReconstructStripeWritePoolLimit = 10 * 3;
+
+  @Config(key = "ec.client.write.pool.limit",
+      defaultValue = "200",
+      description = "Maximum number of threads in the pool for handling client-side" +
+          " write operations in erasure coding. This setting controls the concurrency " +
+          "level for writing data blocks, ensuring efficient data processing and" +
+          " throughput while managing resource utilization.",
+      tags = ConfigTag.CLIENT)
+  private int ecClientWritePoolLimit = 200;
```

Comment:
Since this applies to all writes, not just EC, let's rename it.

Reply:
Done.